audio-worklet/src/bitcrusher-audio-worklet.js

```diff
@@ -61,8 +62,7 @@ registerProcessor('bitcrusher-audio-worklet',
     
     // Add new bit crushed sample to outputBuffer at specified interval.
     for (let j = 0; j < inputChannelData.length; j++) {
-      if (this.index_ % parameters.reduction[j] === 0) {
-        
+      if (this.index_ % Math.round(parameters.reduction[j]) === 0) {
```

Comment:
Add a comment that only integer reduction is supported, hence the rounding.  This should probably also be mentioned in the documentation for the NoiseGate as well.